app/templates/new/components/head.html

```diff
@@ -0,0 +1,46 @@
+<head>
+    <meta charset="utf-8" />
+    <title>{% block pageTitle %}Notify.gov{% endblock %}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Reach people where they are with a text message platform built just for government">
+    <meta name="theme-color" media="(prefers-color-scheme: light)" content="f0f0f0" />
+    <meta name="theme-color" media="(prefers-color-scheme: dark)" content="1b1b1b" />
+    {% if config['NR_MONITOR_ON'] %}
+        {% include "partials/newrelic.html" -%}
+    {% endif %}
+
+    {# Ensure that older IE versions always render with the correct rendering engine #}
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    {% block headIcons %}
+    <link rel="shortcut icon" href="{{ assetPath | default('/assets') }}/images/favicon.ico" />
+    <link rel="icon" type="image/png" sizes="32x32" href="{{ assetPath | default('/assets') }}/images/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="{{ assetPath | default('/assets') }}/images/favicon-16x16.png" />
+    <link rel="mask-icon" href="{{ assetPath | default('/assets') }}/images/safari-pinned-tab.svg" color="#5bbad5">
+    <link rel="apple-touch-icon" sizes="180x180" href="{{ assetPath | default('/assets') }}/images/apple-touch-icon.png">
+    <link rel="manifest" href="/site.webmanifest">
+    <meta name="msapplication-TileColor" content="#da532c">
+    <link href="{{ assetPath | default('/assets') }}/images/notify-dark-favicon.png" rel="icon" media="(prefers-color-scheme: dark)">
+    <meta name="theme-color" content="#ffffff">
+    {% endblock %}
+
+    <link rel="stylesheet" media="screen" href="{{ asset_url('css/styles.css') }}" />
+    {% block extra_stylesheets %}{% endblock %}
+
+    {% block meta_format_detection %}
+    <meta name="format-detection" content="telephone=no">
+    {% endblock %}
+    {% block og_image %}
+    <meta property="og:title" content="Notify.gov">
+    <meta property="og:description" content="Reach people where they are with a text message platform built just for government">
+    <meta property="og:image" content="/static/images/notify-og-image.png">
+    {% endblock %}
+    {# google #}
+    <script type="text/javascript" src="{{ asset_url('js/gtm_head.js') }}"></script>
+    <meta name="google-site-verification" content="niWnSqImOWz6mVQTYqNb5tFK8HaKSB4b3ED4Z9gtUQ0" />
+    {% if g.hide_from_search_engines %}
+    <meta name="robots" content="noindex" />
+    {% endif %}
+
+    {% block head %}{% endblock %}
```

Comment:
Ah I was expecting to see this in the `base.html` template and then all of the content here wrapped with the `block`/`endblock` tags, but I see how this could be helpful in this capacity as well - anything else that would go in the head would go inside the actual `head` tags. 🙂

Reply:
@jonathanbobel and I caught up about this and cleared up some confusion - including is what we want to do here and all is right for now. 🙂